code/__DEFINES/mobs.dm

```diff
@@ -538,6 +538,12 @@ GLOBAL_LIST_INIT(xenoupgradetiers, list(XENO_UPGRADE_BASETYPE, XENO_UPGRADE_INVA
 #define SHRIKE_FLAG_PAIN_HUD_ON		(1<<0)
 #define SHRIKE_CURE_HEAL_MULTIPLIER	10
 
+//Runner defines
+#define RUNNER_EVASION_STACKS				10 //How many projectiles we auto-dodge
+#define RUNNER_EVASION_DURATION				10 SECONDS //How long Evasion lasts.
+#define RUNNER_EVASION_RUN_DELAY			0.5 SECONDS //If the time since the Runner last moved is equal to or greater than this, its Evasion ends.
+#define RUNNER_EVASION_DANGER_THRESHOLD		5 //If we have this number of dodges remaining or fewer, warn the user.
```

Comment:
Fun indentation time. Please make sure these are on the same indent.

Reply:
That's weird, the indent looks fine on my end.

Reply:
github doesnt display this kind of indentation properly